nikola/utils.py

```diff
@@ -2097,3 +2098,12 @@ def rss_writer(rss_obj, output_path):
         if isinstance(data, bytes_str):
             data = data.decode('utf-8')
         rss_file.write(data)
+
+
+def map_metadata(meta, key, config):
+    """Map metadata from other platforms to Nikola names.
+
+    This uses the METADATA_MAPPING setting (via ``config``) and modifies the dict in place."""
+    for foreign, ours in config['METADATA_MAPPING'].get(key, {}).items():
```

Comment:
`config` can be `None`, when `_get_metadata_from_file` is called without `config` specified.